sunpy/conftest.py

```diff
@@ -35,6 +36,15 @@
     HAVE_REMOTEDATA = remotedata_spec is not None
 
 
+def pytest_addoption(parser):
+    parser.addoption("--figure_dir", action="store", default="./figure_test_images")
+
+
+@pytest.fixture(scope='session', autouse=True)
+def figure_base_dir(request):
+    sunpy.tests.helpers.figure_base_dir = request.config.getoption("--figure_dir")
```

Comment:
This is pretty hacky. I for all my searching could not find a better way however.